app/views/filters/filter_controls/_section_form_fields.haml

```diff
@@ -2,4 +2,4 @@
   = section&.title
 - section.controls.each do |control|
   - next unless control
-  = render control.input_partial_path, control: control, f: f
+  = render control.input_partial_path, control: control, f: f, data: (local_assigns[:data][control.id.to_sym] if local_assigns[:data].present?)
```

Comment:
is this only including the partial if `local_assigns[:data].present?` or is it only sending `data: ...` if `local_assigns[:data].present?`?

Might be easier to read as `data: local_assigns[:data].try(&:[], control&.id&.to_sym)`

Reply:
It's only sending `data` if it is present in local_assigns. That's cleaner, I'll update to your suggestion